pkgs/servers/nextcloud/intermediate.nix

```diff
@@ -0,0 +1,25 @@
+{ nextcloud, fetchurl }:
+
+let
+  override = nextcloud.overrideAttrs (old: rec {
+    version = "17.0.2";
```

Comment:
Umm, do we have a way to let the ryantm bot ignore this?

Reply:
Good point.
cc @ryantm in case this is getting merged, could you please blacklist this package for r-ryantm?

Reply:
Sure, I can blacklist the top-level nextcloud attrpath from being updated. Would that cover it for you?

Reply:
The only thing to blacklist would be `nextcloud-intermediate`, but that should be sufficient - thanks!. Please wait with that change until this PR is merged though, this is basically a suggestion how to fix the overall issue and things in here are subject to change.